include/deal.II/lac/affine_constraints.templates.h

```diff
@@ -708,7 +708,8 @@ AffineConstraints<number>::make_consistent_in_parallel(
 
   Assert(sorted == false, ExcMatrixIsClosed());
 
-  Assert(this->local_lines.is_empty() == false,
+  Assert(constraints_to_make_consistent_.is_empty() ==
+           this->local_lines.is_empty(),
```

Comment:
But what if we are currently not storing constraints for any DoFs (e.g., because the current process has no locally owned DoFs -- `local_lines.is_empty()` then returns `true`) but we are *asking* for constraints that are currently stored on other processes for example because we still need to know about these for some reason?

Reply:
Or did you mean `.size()>0` instead of `.is_empty()`?

Reply:
Yes, I wanted `this->local_lines` to be properly initialized, so asking for `size() > 0` is the way to go.